### DISCUSSION TITLE
Store data that in hidden input

### DISCUSSION THREAD

**Discussion by Ahmad-Fathy** [filamentphp/filament#5636] [General]
I have a form input for "slug" column which auto generated from` user name` field
```
        TextInput::make('user_name')
          ->rules(['required', 'alpha_spaces', 'min:4', 'max:50', new ArabicRule()])
          ->unique(ignoreRecord: true)
          ->label(__('site.user_name'))
          ->reactive()
          ->afterStateUpdated(fn (Closure $set, $state) =>
            $set('slug', Str::slug($state))
          ),
```
 .. so i want to hide it .. but when i use `hidden()` attribute the `slug `value did not saved

**Comment by danharrin** [comment]
use a Hidden field instead of a TextInput with hidden()

**Reply by BenceBakos** [reply]
What about More complex input fields, like repeaters?

**Reply by leandrocfe** [reply]
It would be the same. What are you trying to do?
